pkg/test/framework/features/features.yaml

```diff
@@ -30,9 +30,16 @@ features:
   # features that allow users to secure their services and service mesh.
   security:
     peer:
+      secure-naming:
     user:
     control-plane:
       k8s-certs:
+      mtls:
+      tls:
+      plugin-cert:
+    webhook:
```

Comment:
This is really vague.. it could use a comment. Same with the rest

Reply:
webhook, chiron and vault are all not needed anymore